bigbone/src/main/kotlin/social/bigbone/api/method/FilterMethods.kt

```diff
@@ -59,6 +60,11 @@ class FilterMethods(private val client: MastodonClient) {
         expiresIn: Int? = null,
         filterAction: Filter.FilterAction = Filter.FilterAction.WARN
     ): MastodonRequest<Filter> {
+        // see: https://github.com/mastodon/mastodon/pull/34256
+        if (filterAction == Filter.FilterAction.BLUR && client.getInstance().apiVersions.mastodon < 5) {
+            throw BigBoneRequestException("FilterAction.BLUR not available on Mastodon instance with API version < 5")
```

Comment:
We might want to add a more specific exception type but I’m fine with only doing that once we have a better way of handling such checks without having to add what you just added here to every request and type.